### PR TITLE
Add question on Windows

### PR DESCRIPTION
**Pull request by juli1**
Clarify our support for WIndows.

### CODE REVIEW

File: content/en/code_analysis/faq/_index.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ The analyzer is periodically checked against the benchmark and updated results a
 
 Datadog's static analyzer is available under an open source license and the code is [available on GitHub][4].
 
+### Does the analyzer runs on Windows?
+
+Currently, the Datadog Static Analyzer only runs on UNIX-based system. We have an open feature request
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:79]:
```suggestion
Currently, the Datadog Static Analyzer only runs on UNIX-based systems. We have an open feature request
```

File: content/en/code_analysis/faq/_index.md [hunk 2]
```diff
@@ -74,6 +74,11 @@ The analyzer is periodically checked against the benchmark and updated results a
...
+to support Windows that is [trackable on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:80]:
```suggestion
to support Windows that can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```

File: content/en/code_analysis/faq/_index.md [hunk 3]
```diff
@@ -74,6 +74,11 @@ The analyzer is periodically checked against the benchmark and updated results a
 
 Datadog's static analyzer is available under an open source license and the code is [available on GitHub][4].
 
+### Does the analyzer runs on Windows?
+
+The Datadog Static Analyzer only runs on UNIX-based systems. An open feature request
+to support Windows that can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:80]: My bad, forgot to remove `that` here.
```suggestion
to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```

File: content/en/code_analysis/faq/_index.md [hunk 4]
```diff
@@ -74,6 +74,11 @@ The analyzer is periodically checked against the benchmark and updated results a
 
 Datadog's static analyzer is available under an open source license and the code is [available on GitHub][4].
 
+### Does the analyzer runs on Windows?
+
+The Datadog Static Analyzer only runs on UNIX-based systems. An open feature request
+to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```
Comment (neko-dd) [inline content/en/code_analysis/faq/_index.md:80]:
```suggestion
The Datadog Static Analyzer runs only on UNIX-based systems. An open feature request to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```

Comment (neko-dd) [reply content/en/code_analysis/faq/_index.md:80]: small grammar nit

**Review by neko-dd — APPROVED** [review]

**Review by amaanq — APPROVED** [review]

**Pull request merged**